cmd/zetatool/inbound/bitcoin.go

```diff
@@ -0,0 +1,214 @@
+package inbound
+
+import (
+	"context"
+	"encoding/hex"
+	"fmt"
+	"math/big"
+
+	cosmosmath "cosmossdk.io/math"
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/rpcclient"
+	"github.com/rs/zerolog"
+
+	"github.com/zeta-chain/node/cmd/zetatool/config"
+	"github.com/zeta-chain/node/pkg/chains"
+	"github.com/zeta-chain/node/pkg/coin"
+	"github.com/zeta-chain/node/pkg/rpc"
+	crosschaintypes "github.com/zeta-chain/node/x/crosschain/types"
+	"github.com/zeta-chain/node/x/observer/types"
+	"github.com/zeta-chain/node/zetaclient/chains/bitcoin/common"
+	zetaclientObserver "github.com/zeta-chain/node/zetaclient/chains/bitcoin/observer"
+)
+
+func btcInboundBallotIdentifier(
+	cfg config.Config,
+	zetacoreClient rpc.Clients,
+	inboundHash string,
+	inboundChain chains.Chain,
+	zetaChainID int64) (string, error) {
```

Comment:
I think this comes up a lot for naming 
ChainID vs ChainId 
We have both in the codebase; I lean more towards ChainID when defining variable names, please comment on this thread if you disagree

Reply:
`foreignChainID` would be my preference when referring to foreign chains.

But it's hard to find the right thing because we have our own internal IDs too. `zetaForeignChainID` in that case?

Reply:
In this case, this is the zetachainID, so it's fine.
In general, though, we are starting to use connected chains as the terminology

Reply:
Yes, Connected Chain for chain we connect to. Foreign Chain terminology needs to be removed from the codebase